dictionary.txt

```diff
@@ -0,0 +1,7 @@
+achiever
+adversary
+affix
```

Comment:
I like that you put your list of words in a separate file! Making a distinction between logic and data is an important part of programming, and we'll be talking about this theme more as the class goes on.